www/js/emulator.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 
 this.emulator = new Object();
+// This is technically race-y, but should be fine for now.
+Binjgb().then(v => globalThis.Module = v);

```

Comment:
Why is this needed? Is it related to the changes introduced by the binjgb fork? IIRC locally, when I use the binjgb build currently in the `gh-pages` branch I don't need to make this modification here

Reply:
I am using daid’s fork for now in this PR. I wonder if I have a new Emscripten or something? The Wasm build of binjgb only defines a `Binjgb` global factory, that _returns_ a `Module` asynchronously (which is the pattern I’m used to with Emscripten when [`-s MODULARIZE=1`](https://emsettings.surma.technology/#MODULARIZE) is set). 

I have no idea how it was previously built to skip this... I guess I could see if I can unset `MODULARIZE` somehow...